static/js/components/CandidateCommentList.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Candidate comment list - different layout / functionality than the source comment list, so it gets its own component